docs/understanding-airbyte/supported-data-types.md

```diff
@@ -24,7 +24,7 @@ This table summarizes the available types. See the [Specific Types](#specific-ty
 | Object                                                         | `{"type": "object"}`; optionally `properties` and `additionalProperties`                 | `{"foo": "bar"}`                                                                |
 | Untyped (i.e. any value is valid)                              | `{}`                                                                                     |                                                                                 |
 | Union                                                          | `{"anyOf": [...]}` or `{"oneOf": [...]}`                                                 |                                                                                 |
-| Boolean                              | `{}`                                                                                     |                                                                                 |
+| Boolean                              | `{"type": "boolean"}`                                                                                     |                                `True` or` False`                                                 |
```

Comment:
I did what I thought the boolean type would be. I did not have a reference to an API or anything that takes this data.